helm/rbac-app/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+appVersion: 0.0.1
+annotations:
+  application.giantswarm.io/team: "rainbow"
+  config.giantswarm.io/version: 1.x.x
+name: rbac-app
+description: A Helm chart for rbac
+engine: gotpl
+home: https://github.com/giantswarm/rbac-app
+version: 0.1.0 
```

Comment:
You either need to use the architect templates `[[ ]]` or switch to `executor: app-build-suite` in the circle-ci config